src/WebJobs.Script/Rpc/ProcessManagement/IProcessRegistry.cs

```diff
@@ -1,11 +1,12 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using System;
 using System.Diagnostics;
 
 namespace Microsoft.Azure.WebJobs.Script.Rpc
 {
-    internal interface IProcessRegistry
+    internal interface IProcessRegistry : IDisposable
```

Comment:
Any reason why we're pushing this to the interface? Concrete implementations should opt to implement `IDisposable`, if needed. The empty registry above, for example, wouldn't need to implement it.

Reply:
Thanks for catching this. This change is not needed.